src/test/rpc/LedgerRPC_test.cpp

```diff
@@ -1220,21 +1221,28 @@ class LedgerRPC_test : public beast::unit_test::suite
     }
 
     void
-    testLedgerEntryUnknownOption()
+    testLedgerEntryInvalidParams(unsigned int apiVersion)
     {
-        testcase("ledger_entry Request Unknown Option");
+        testcase(
+            "ledger_entry Request With Invalid Parameters v" +
+            std::to_string(apiVersion));
         using namespace test::jtx;
         Env env{*this};
 
         std::string const ledgerHash{to_string(env.closed()->info().hash)};
 
         // "features" is not an option supported by ledger_entry.
         Json::Value jvParams;
+        jvParams[jss::api_version] = apiVersion;
         jvParams[jss::features] = ledgerHash;
         jvParams[jss::ledger_hash] = ledgerHash;
         Json::Value const jrr =
             env.rpc("json", "ledger_entry", to_string(jvParams))[jss::result];
-        checkErrorValue(jrr, "unknownOption", "");
+
+        if (apiVersion < 2)
```

Comment:
Consider 2u (unsigned int) instead of 2 (signed int)

Reply:
Good catch. thank you!